apps/wallet-mobile/src/components/AmountItem/AmountItem.tsx

```diff
@@ -43,6 +46,7 @@ export const AmountItem = ({
   const formattedQuantity = Quantities.format(quantity, tokenInfo.decimals ?? 0)
   const showSwapDetails = !isPrimary && variant === 'swap'
   const priceImpactRiskTheme = usePriceImpactRiskTheme(priceImpactRisk ?? 'none')
+  const priceImpactRiskTextColor = orderType === 'market' ? priceImpactRiskTheme.text : styles.text.color
```

Comment:
hey @SorinC6 just to confirm, so it won't hide, it will just display without emphasis in the color?

Reply:
I had a task before for when is a limit order to not display the Price Impact on the Confirm Transaction(summary) screen. As I see it if there is a limit order everything will be as usual and not show the price impact colors and warnings 🤔